lib/stack_master/parameter_resolvers/stack_output.rb

```diff
@@ -10,14 +10,14 @@ def initialize(config, stack_definition)
         @config = config
         @stack_definition = stack_definition
         @stacks = {}
+        @output_regex = %r{(?:([^:]+):)?([^:/]+)/(.+)}
       end
 
       def resolve(value)
-        validate_value!(value)
-        stack_name, output_name = value.split('/')
-        stack = find_stack(stack_name)
+        region, stack_name, output_name = parse!(value)
+        stack = find_stack(stack_name, region)
         if stack
-          output = stack.outputs.find { |output| output.output_key == output_name.camelize }
+          output = stack.outputs.find { |o| o.output_key == output_name.camelize }
```

Comment:
I prefer using readable variables over shorter ones so that the code flow isn't interrupted by me trying to remember what the shorter ones actually mean.